R/metab_simple.R

```diff
@@ -24,7 +24,9 @@
 #'   
 #' @author Alison Appling, Jordan Read; modeled on lakeMetabolizer
 #' @examples
-#'  metab_simple(data=data.frame(date.time=rep(as.Date("2015-04-15"), 24), DO.obs=1:24, DO.sat=rep(14, 24), PAR=sin((1:24)*pi/24)^8, temp.water=15))
+#'  metab_simple(data=data.frame(
```

Comment:
Ok, but a lot of use cases assume salinity~0, and use altitude for baro, so since wtr.temp and do.obs are already there, it doesn't seem onerous to support 
```R
metab_simple(data = data.frame(date.time=rep(as.Date("2015-04-15"), 24), DO.obs=1:24, temp.water=15), altitude=424)
# as input to 
metab_simple(data, ...)
```